flask_mysqldb/__init__.py

```diff
@@ -78,7 +79,7 @@ def connect(self):
         if current_app.config['MYSQL_CURSORCLASS']:
             kwargs['cursorclass'] = getattr(cursors, current_app.config['MYSQL_CURSORCLASS'])
 
-        return MySQLdb.connect(**kwargs)
+        return MySQLdb.connect(**kwargs, autocommit=current_app.config['MYSQL_AUTOCOMMIT'])
```

Comment:
@shaunpud would it be an idea to follow the original pattern and add the autocommit var to the connect function?

Something like this:
'''
      if current_app.config['MYSQL_AUTOCOMMIT']:
            kwargs['....'] = current_app.config['...']
'''

Reply:
Yep, good one 👍